tests/integration_test_dag.py

```diff
@@ -130,4 +130,5 @@ def test_full_dag(database_table_fixture, sample_dag):
             output_file=File(path="/tmp/out_agg.csv"),
             if_exists="replace",
         )
+        aql.cleanup()
```

Comment:
@dimberman what does this look like in the DAG UI? Do we see the task & all the tasks on which it depends on?

Reply:
@tatiana it looks like this. The task will run in parallel and had no "airflow based" dependencies.
<img width="757" alt="Screen Shot 2022-06-16 at 10 25 18 AM" src="https://user-images.githubusercontent.com/2644098/174130528-b468417f-fbbe-45c6-8265-49dffc1076f4.png">